src/task.rs

```diff
@@ -2407,3 +2393,15 @@ impl<'a> Runner<'a> {
         Ok(all_ok)
     }
 }
+
+pub fn relative_path(name: &String) -> String {
+    if name.starts_with(':') || (!name.contains('/') && !name.contains("\\")) {
+        name.to_owned()
+    } else {
+        diff_paths(Path::new(&name), current_dir().unwrap())
+            .unwrap()
+            .to_str()
+            .unwrap()
+            .to_string()
```

Comment:
It should be equivalent and shorter to write:

```suggestion
        diff_paths(Path::new(&name), current_dir().unwrap())
            .unwrap()
            .to_string_lossy();
```

Reply:
Also, since each task can have its own cwd, should we not also explicitly pass the cwd / base?

Reply:
Where I can find this cwd / base?

Reply:
`cwd` is a property on all tasks. I think you are already handling this in the task absolute pathing side?